### ISSUE TITLE
Add Search possibility when selecting the From/To column for relationships

### ISSUE THREAD

**Issue by SalCag** [otykier/TabularEditor#778]
It is a tedious task to create a new relationship in the model of a customer with over 230 tables and thousands of columns. I always need to scroll up and down to find the correct column:
![image](https://user-images.githubusercontent.com/26252452/109144343-59f9ac80-7761-11eb-9f3f-6235291ed85c.png)

I would love to have a Search field in the Dropdown to be able to enter the needed column

Alternatively, I would like to be able to start entering the table name in the "From column" or "To column" field, and the drop-down starts at the correct position.
Or a Drag-And-Drop functionality from the column list in the "From column" and "To column" would be a nice addition as well.

I think that the search option would be the easiest to implement and use.

I'm looking forward to seeing this feature in an upcoming release, as it would make life much easier.

Regards
Salvatore

**Comment by otykier** [comment]
This has been raised before (#372). Are you aware that you can create a relationship directly from or to any column by right-clicking on that column and then navigating to the destination table and column:

![image](https://user-images.githubusercontent.com/8976200/109146014-8f9f9500-7763-11eb-8208-de4a88f0dae5.png)

The menu will only show tables that do not already have a relationship with the table in which the column resides, and it also only shows columns that have the same data type (since it's not possible to create relationships between two columns of different types).

**Comment by SalCag** [comment]
No. I was not aware of this possibility. And I wasn't aware of #372 
This is of enormous help.

Thank you.

Anyway, I still think that a Search field in the Dropdown list could still be a good idea.
